integration/utils/contLoadGenerator.go

```diff
@@ -386,81 +401,152 @@ func (lg *ContLoadGenerator) sendAddKeyTx(workerID int) {
 
 	err = addKeysTx.AddArgument(cadenceKeysArray)
 	if err != nil {
-		lg.log.Error().Err(err).Msgf("error constructing add keys to account transaction")
+		lg.log.Error().Err(err).Msgf("workerID=%d error constructing add keys to account transaction", workerID)
 		return
 	}
 
-	lg.log.Trace().Msgf("creating transaction")
+	lg.log.Trace().Msgf("workerID=%d creating transaction", workerID)
 
 	addKeysTx.SetReferenceBlockID(blockRef).
 		SetProposalKey(*acc.address, 0, acc.seqNumber).
 		SetPayer(*acc.address).
 		AddAuthorizer(*acc.address)
 
-	lg.log.Trace().Msgf("signing transaction")
+	lg.log.Trace().Msgf("workerID=%d signing transaction", workerID)
 	err = acc.signTx(addKeysTx, 0)
 	if err != nil {
-		lg.log.Error().Err(err).Msgf("error signing transaction")
+		lg.log.Error().Err(err).Msgf("workerID=%d error signing transaction", workerID)
 		return
 	}
 
-	lg.sendTx(addKeysTx)
+	lg.sendTx(workerID, addKeysTx)
+}
+
+func (lg *ContLoadGenerator) pushAccountsHappening(workerID int, acc *flowAccount, tx_id string, timeNow time.Time) {
+	lg.log.Trace().Msgf("workerID=%d pushing happening address", workerID)
+	lg.happeningAccounts <- (func() (*flowAccount, string, time.Time) { return acc, tx_id, timeNow })
+}
+
+func (lg *ContLoadGenerator) probeAccountsHappening(workerID int) {
+	channelElements := len(lg.happeningAccounts)
+	lg.log.Trace().Msgf("workerID=%d probing happening address from channel with %d elements", workerID, channelElements)
+	for i := 0; i < channelElements; i++ {
+		acc, tx_id, timeAtSendTx := (<-lg.happeningAccounts)()
+		txIdFile := fmt.Sprintf("/dev/shm/flow-transaction-feedback/%s/%s/%s.txt", tx_id[0:2], tx_id[2:4], tx_id[4:]) // e.g. /dev/shm/flow-transaction-feedback/70/b9/4146a01bc7843dc9547837e19ba0da6899fdd83ae61e2cf2e3fb4555da6a.txt
+		lg.log.Trace().Msgf("workerID=%d probing happening address %x account %d tx_id %s AKA %s", workerID, acc.address.Bytes(), acc.i, tx_id, txIdFile)
+		if _, err:= os.Stat(txIdFile); errors.Is(err, fs.ErrNotExist) {
+			// come here if file does not exist; means account is still happening...
+			lg.pushAccountsHappening(workerID, acc, tx_id, timeAtSendTx)
+
+			elapsed := time.Since(timeAtSendTx).Seconds()
+			elapsedSuspicious := 30.0
+			if (elapsed >= elapsedSuspicious) && (elapsed < (1+elapsedSuspicious)) {
+				// come here to capture transactions which mysteriously have had no feedback after 30 seconds
+				// note: this should only be output once as its age reaches and passes through a certain amount of seconds...
+				lg.log.Warn().Msgf("workerID=%d address %x account %d tx_id=%s has been happening for %f seconds and counting", workerID, acc.address.Bytes(), acc.i, tx_id, elapsed)
+			}
+		} else {
+			// come here if file exists; means account is good to be used again
+			lg.availableAccounts <- acc
+			e := os.Remove(txIdFile)
```

Comment:
If you take my suggestion above, for `isAccountReady`, will will either have to remove this (e.g. just leave garbage there, since the txIDs should always be unique) or add a `cleanup` function that takes the tx id.